### DISCUSSION TITLE
👋 Welcome!

### DISCUSSION THREAD

**Discussion by YixinKristy** [PaddlePaddle/PaddleDetection#6179] [Announcements]
We’re using Discussions as a place to connect with other members of PaddleDetection. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PengKunGit** [comment]
当用paddledetection中的车牌检测时。车头有文字的时候，没有检测到车牌，而是检测到文字上了，想问问能怎么解决!
![Uploading OCRv3.png…]()

**Comment by PengKunGit** [comment]
<img width="948" alt="OCRv3" src="https://user-images.githubusercontent.com/117342806/210023263-c231292c-df4a-44ad-ae7f-d8164a3a75e3.png">
